modules/server/src/network/resolvers/connections/index.ts

```diff
@@ -0,0 +1,70 @@
+import { NetworkAggregationConfig, NetworkAggregationConfigInput } from '@/network/types';
+import { ObjectValues } from '@/utils/types';
+import axios from 'axios';
+import urljoin from 'url-join';
+
+/**
+ * Connection status types
+ */
```

Comment:
This description provides no more information than the name of the variable... Possibly not needed.

Reply:
my train of thought: I could see this being moved based on how we build out connection health checks - it's a nice fence of here's the connection types.

Just helps. More comment blocks. I don't think it confuses. So no negative impact. Bigger fish to fry and overall we want more docs not less 👍